src/app/features/itineraries/components/pages/list/ListItinerariesPage.tsx

```diff
@@ -20,7 +20,7 @@ const ListItinerariesPage: React.FC = () => {
   }, [data])
 
   return <DefaultLayout>
-    { data?.itineraries.length > 1 && (
+    { data?.itineraries.length !== undefined && (
```

Comment:
Looks like a weird typecast. Or is this unknown, external (impure) data? Could this also be `data !== undefined && ` or `data &&`

Reply:
Changed it